internal/expr/prepare.go

```diff
@@ -261,6 +262,7 @@ func (pe *ParsedExpr) Prepare(args ...any) (expr *PreparedExpr, err error) {
 	var inputs = make([]typeMember, 0)
 	var typeMemberPresent = make(map[typeMember]bool)
 	var outputCols = make([][]columnName, 0)
+	sliceRanges := make(map[typeMember]sliceRange)
```

Comment:
Even though its not the advised way to declare new map, this should be consistent with the ones in the same block.